models.js

```diff
@@ -15,6 +15,8 @@ debuglog = function () {
 
 (function() {
     var xpm = xpathmodels;
+    xpm.validHashtagNamespaces = [];
+    xpm.hashtagToXPathDict = {};
```

Comment:
To make the library backward-compatible with the current global `xpathmodels` object, we could set up that global instance with a reject or passthrough hashtag interpreter (not sure which is prefered).